crates/next-dev-tests/tests/integration/next/font-google/basic/pages/index.js

```diff
@@ -67,13 +66,34 @@ async function getRuleMatchingClassName(className) {
       });
     }
 
-    for (const rule of stylesheet.sheet.cssRules) {
-      if (rule.selectorText === selector) {
-        matchingRule = rule;
-        break;
+    const sheet = stylesheet.sheet;
+
+    if (sheet == null) {
```

Comment:
can sheet be null if we've waited for the stylesheet to load above? Should this just be an invariant?

Reply:
This check came back with the merge, but it should no longer be necessary indeed.